shesha/scripts/saxoPlusScriptExample.py

```diff
@@ -4,11 +4,11 @@
 The saxo+ manager can be instanciated using the follwing commands. 
 
 > cd $HOME/compass/shesha/shesha/widgets/
-> ipython -i widget_saxoplus.py ../../data/par/SPHERE+/sphere.py ../../data/par/SPHERE+/sphere+.py 3
+> ipython -i widget_saxoplus.py ../../data/par/SPHERE+/sphere.py ../../data/par/SPHERE+/sphere+.py--freqratio 3
```

Comment:
Should be `ipython -i widget_saxoplus.py ../../data/par/SPHERE+/sphere.py ../../data/par/SPHERE+/sphere+.py -- --freqratio 3`
The first "--" is mandatory with ipython to separate the arguments you give to your application from the ones you can give to ipython itself